stdlib/ballerina-builtin/src/main/ballerina/internal/exec_ballerina.bal

```diff
@@ -24,4 +24,28 @@ documentation{
     R{{}} - Data piped from the standard output and error output of the process
 }
 public native function execBallerina(@sensitive BallerinaCommand command, @sensitive string packagePath)
-    returns (string|error);
\ No newline at end of file
+    returns (string|error);
+
+documentation {
+    Executes a ballerina doc command
+    P{{packageList}} - List of packages
+    P{{outputPath}} - Path to write api-docs
+    P{{templatesPath}} - Location of the templates
+    P{{exclude}} - Packages to exclude
+    P{{includeNatives}} - Generate docs for natives
+    P{{envVars}} - Environment variables to pass
+    P{{config}} - Path to configuration file
+    P{{verbose}} - Verbose mode
+    P{{sourceRoot}} - Root folder of the packages
+    R{{}} - Data piped from the standard output and error output of the process
+}
+public native function execBallerinaDoc(string[] packageList,
+                                        string? sourceRoot = (),
+                                        string? outputPath = (),
+                                        string? templatesPath = (),
+                                        string[]? exclude = (),
+                                        boolean includeNatives = false,
+                                        map<string>? envVars = (),
+                                        string? config = (),
+                                        boolean verbose = false)
+                           returns (string|error);
```

Comment:
Add a new line?

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-lang/pull/9227/commits/9b2f2ce167828ea857fabb067b03d97ac5c4f139